test/.envrc-example

```diff
@@ -25,10 +25,11 @@ export PARACHAIN_ID=1000
 
 # export ETH_RPC_ENDPOINT=https://ropsten.infura.io/v3
 # export ETH_WS_ENDPOINT=wss://ropsten.infura.io/ws/v3
- 
+
 # export ROPSTEN_PRIVATE_KEY=         # Your deployment account private key
 # export BEEFY_RELAY_ETH_KEY=         # Your Beefy relayer account private key
 # export PARACHAIN_RELAY_ETH_KEY=     # Your Parachain relayer account private key
 # export E2E_TEST_ETH_KEY=            # Your E2E test account private key
 
-#export START_BEACON_SYNC=false       # Start the Beacon chain syncing (in development)
\ No newline at end of file
+# export START_BEACON_SYNC=false       # Start the Beacon chain syncing (in development)
+# export INITIAL_BEACON_BLOCK=0x088241fcf1cf63040b804498c945d3a6ae5484e65692483747fba5b8902c99c9 # Initial block to be synced from (this value is on ropsten)
```

Comment:
Can we get the initial beacon block using a script? We do this with `beefy`.

Reply:
We could just get the latest block, yeah. However, the light client protocol suggests always syncing from a manually validated block, so that you know you are syncing on the right chain. Is that not also an issue with Beefy?

Reply:
Sorry maybe beefy is the wrong example here. In the `ethereum-light-client` we know the block is trusted by getting the latest block from a trusted node (Infura) as opposed to setting an environment variable. Can we not do something similar for beacon? Fetch the latest sync committee from a trusted beacon node at time of building the genesis state?

Reply:
I can get the latest finalized block from a trusted node, yeah! Let me do that quick.

I think the sync committee fetching can stay in the relayer for now - I'll relook at it once I get to the [resuming syncing ticket](https://linear.app/snowfork/issue/SNO-292/relayer-and-beacon-client-should-support-syncing-from-last-synced-slot).

Reply:
Added in https://github.com/Snowfork/snowbridge/pull/658/commits/31bb446bed0110d837aaa3430b498ca1fccb08bb.